### PR TITLE
Fixes #91 Use requirements.txt to install dependencies in pip install

### LINKED ISSUE #91: Use requirements.txt to install dependencies in pip install
**Current Behavior:**
Currently, `install_requires` is used to install dependencies.
```
install_requires=[
        "requests==2.16.0",
        "xmljson==0.1.9"
],
```

**Expected Behaivour:**

Use `requirements` file to install dependencies directly, Without specifying the future dependencies in `setup.py`
In future, if some deps comes then it will be parsed using `requirements` file and do not require to add again in `install_requires`

**Yes, wanna work on this issue**

### PR DESCRIPTION
**Pull request by bhaveshAn**
Add `MANIFEST.in` and install dependencies directly from `requirements.txt`
Added `README.rst` which will be parsed as the long description in the PyPi package page.
Fixed bug  of `download_url` and `url` values.

Fixes #91 

@singhpratyush @sudheesh001 
Please review the entries at `README.rst` (I have used `pandoc` utility to convert `md` to `rst` directly). 
Parsed rst is at https://github.com/bhaveshAn/loklak_python_api/blob/require/README.rst
Thanks !!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+
+| If you want to set this API to use your own server, you can now define
+  it by doing
+| ``l = Loklak('http://192.168.192.5:9000/')`` for example or pass a URL
+  to it as
+| ``l = Loklak('http://loklak-super-cluster.mybluemix.net/')``
+
+Note the trailing ``/`` is important and so is ``http://``
+
+API Documentation
```
Comment (daminisatya) [inline README.rst]: @sudheesh001 @bhaveshAn Do you think we should have a separate wiki page or an MD file for the API documentation?

Comment (bhaveshAn) [reply README.rst]: Yes, I think creating a wiki page for API Docs and then referencing it through `README.rst`
would be nice way to go.

**Comment by vaibhavsingh97** [comment]
Ok, I restarted the build and it passed. @bhaveshAn can you help me in figuring out why these builds suddenly stopped.

File: README.rst (hunk 1 above)
Comment (sudheesh001) [reply README.rst]: The problem with `wiki` pages is that they can quickly go out of sync with the rest of the codebase. Maybe generate a `readthedocs` style of documentation for the project?

**Comment by bhaveshAn** [comment]
@sudheesh001 Can you please merge it .

**Comment by sudheesh001** [comment]
Yes that would be nice.

**Pull request merged**